modules/ROOT/pages/producing-ign.adoc

```diff
@@ -51,7 +51,7 @@ An easy way to use `fcct` is to run it in a container with `podman`:
 +
```

Comment:
Optional: a link back to https://github.com/coreos/fcct/pull/70 in this commit's message for posterity would be good.

(Grrr... on GitHub for not allowing commenting on a commit message. I'm talking about 8f4c3f200768662a61d0ef3481d1438d780fe25c here.)